### PR TITLE
Add TWIA issues 4, 5, and 7

### PR DESCRIPTION
**Pull request by ebkalderon**
This pull request modifies issue #5. For these newly added _TWIA_ posts, I have used the proper date format specified in pull request #17) to make sure they are sorted correctly.

### CODE REVIEW

File: src/_posts/twia-5.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+extends: post.tpl
+
+title: This Week in Amethyst 5
+date: 15 February 2016 13:04 -0500
```
Comment (LucioFranco) [inline src/_posts/twia-5.md]: It must be `15 February 2016 13:04:00 -0500`

File: src/_posts/twia-7.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+extends: post.tpl
+
+title: These Weeks in Amethyst 7
+date: 13 March 2016 16:23 -0500
```
Comment (LucioFranco) [inline src/_posts/twia-7.md]: It must be `13 March 2016 16:23:00 -0500`

**Comment by ebkalderon** [comment]
The checks passed and the date/time formats seem correct, so I think this is ready to merge. Thoughts, @LucioFranco?

**Comment by LucioFranco** [comment]
Looks good to me! I think it's ready to be merged! @ebkalderon

**Pull request merged**